### PR TITLE
Added more fucking

### PR DESCRIPTION
**Pull request by sniff122**
Added more fucking to the HTML.

### CODE REVIEW

**Comment by wtfismyip** [comment]
Looks fucking good

**Comment by sniff122** [comment]
Fucking amazing!

File: wtf/static/static/html.template [hunk 1]
```diff
@@ -102,11 +102,11 @@ a:link,a:visited,#tor{color:#f30}
             <p class="blah"><a href="https://github.com/wtfismyip/wtfismyip">GitHub</a></p>
          </li>
          <li>
-            <p class="blah"><a href="/donate">Donate</a></p>
+            <p class="blah"><a href="/donate">Fucking Donate</a></p>
```
Comment (cgaylord-vt) [inline wtf/static/static/html.template:105]: How about "Just Fucking Donate Already!"

File: wtf/static/static/html.template [hunk 2]
```diff
@@ -102,11 +102,11 @@ a:link,a:visited,#tor{color:#f30}
...
          </li>
       </ul>
       <ul>
-         <li>Don't like being tracked? Download the <a href="https://www.torproject.org/download/">Tor Browser Bundle!</a>
+         <li>Don't like being tracked? Download the fucking <a href="https://www.torproject.org/download/">Tor Browser Bundle!</a>
```
Comment (cgaylord-vt) [inline wtf/static/static/html.template:109]: *fucking tracked...

**Review by cgaylord-vt — COMMENTED** [review]
Too much fucking going on... :-)

**Comment by sniff122** [comment]
You can never have too much fucking

**Comment by cgaylord-vt** [comment]
Fucking embarrassing

**Pull request merged**